conan/tools/cmake/cmake.py

```diff
@@ -254,3 +255,14 @@ def _cmake_log_levels_args(self):
         if log_level == "quiet":
             log_level = "error"
         return ["--loglevel=" + log_level.upper()] if log_level is not None else []
+
+    def _run(self, command, redirect_stdout=None, **kwargs):
+        if redirect_stdout is not None:
+            with open(redirect_stdout, "w") as f:
```

Comment:
this is overwriting, not appending log, isn't it? Is this the most expected behavior, or users passing the same file to 3 configure(), build() and install() will expect the logs to append?